README.md

```diff
@@ -49,6 +49,11 @@ ch_cookie_consent:
         - 'marketing'
         - 'social_media'
     use_logger: true # Logs user actions to database
+    
+    # add trying
```

Comment:
There is no explanation whatsoever about why 'neo' is here, you should explain what these settings do, why they are possible/needed and what they default to.

Reply:
"neo" was for my internal testing.